modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb

```diff
@@ -30,18 +30,50 @@ class MissingUserAttributesError < StandardError; end
     end
 
     def perform(submission_id, user_uuid)
-      submission = DebtsApi::V0::Form5655Submission.find(submission_id)
+      @submission_id = submission_id
+      @user_uuid = user_uuid
+
+      # Try Redis
       user = UserProfileAttributes.find(user_uuid)
```

Comment:
This is not feedback but an observation.

My active record brain had alarm bells go off. `#find` throws an error if no record exists. I checked in rails console and this doesn't!

Reply:
Yep, just Redis just returns nil where AR would throw